src/components/Tabs/BarItem.js

```diff
@@ -1,37 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/react';
-import {
-  useTessen,
-  useInstrumentedData,
-} from '@newrelic/gatsby-theme-newrelic';
 import useTabs from './useTabs';
 
-const BarItem = ({ index, children, id, count, disabled }) => {
+const BarItem = ({ index, children, id, count, disabled, ...props }) => {
```

Comment:
You can totally rename during deconstruction!
```suggestion
const BarItem = ({ index, children, id, count, disabled, onClick: onTabClick }) => {
```